app/src/main/java/org/p2p/wallet/settings/ui/settings/NewSettingsPresenter.kt

```diff
@@ -60,15 +60,14 @@ class NewSettingsPresenter(
 
     override fun changeZeroBalanceHiddenFlag(hideValue: Boolean) {
         settingsInteractor.setZeroBalanceHidden(hideValue)
-        loadSettings()
     }
 
     override fun onBiometricSignInSwitchChanged(isSwitched: Boolean) {
         if (isSwitched) {
             view?.confirmBiometrics(authInteractor.getPinEncodeCipher())
         } else {
             authInteractor.disableBiometricSignIn()
-            loadSettings()
+            view?.updateSwitchItem(R.string.settings_item_title_touch_id, isSwitched = false)
```

Comment:
load settings should get the current value of the biometric and update all the switches accordingly 
it doesn't work as expected?